### DISCUSSION TITLE
how do I attach existing couchdb /var/lib/couchdb to a new install?

### DISCUSSION THREAD

**Discussion by AtlantaKid** [apache/couchdb#4025] [General] [answered]
Hi, I am running Ubuntu 20.x with CouchDB 3.2 and my server OS got corrupted and I had to do a fresh install of OS + CouchDB, I have copy of my old /var/lib/couchdb folder, how do I copy that in and make the new install of couchdb see it?

**Comment by nickva** [comment, marked as answer]
You can copy it over mostly see: https://docs.couchdb.org/en/stable/maintenance/backups.html

If you want to try it out first, you can mount in a different location first then configure CouchDB's [database_dir](https://docs.couchdb.org/en/stable/config/couchdb.html#couchdb/database_dir) and [view_index_dir](https://docs.couchdb.org/en/stable/config/couchdb.html#couchdb/view_index_dir).

You may also want to restore the `etc` config directory. It would have various settings such as the [uuid](https://docs.couchdb.org/en/stable/config/couchdb.html#couchdb/uuid) admin users' password hashes and anything else you might have set.

Especially for a clustered install, pay attention to use  the same node name in `vm.args` as the shard map which maps db nodes and shard ranges encodes. See https://docs.couchdb.org/en/stable/setup/cluster.html. If it's just a single node install, this step might be easier if you have something like `couchdb@127.0.0.1`.

**Reply by AtlantaKid** [reply]
Unfortunately I do not have a backup of couchdb config files, but I created the same linux userid and password I used before and placed those in /opt/couchdb/etc/local.ini the only backup I have is from /var/lib/couchdb, 
looking at /opt/couchdb/etc/default.ini I see 
```
 uuid =
database_dir = ./data
view_index_dir = ./data
```
should I use /var/lib/couchdb in place of ./data ?  what value do I need for uuid ?

Please advise.

**Reply by AtlantaKid** [reply]
Got it working -- Thank you so so much for your advise, I snapshot my Linux instance just to be safe and updated the /opt/couchdb/etc/default.ini entries for database_dir  and  view_index_dir  and changed it /var/lib/couchdb and the web/gui with "Project Foxton - database" worked fine, I am able to see my previously configured databases and their content.

**Reply by nickva** [reply]
No problem. Thanks for reaching, out!
